src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImplGetCaseByIdTest.java

```diff
@@ -238,4 +319,34 @@ private void verifyCase(
   private List<CaseContainerDTO> casesFromCaseService() {
     return FixtureHelper.loadPackageFixtures(CaseContainerDTO[].class);
   }
+
+  private Date utcDate(LocalDateTime dateTime) {
+    return Date.from(dateTime.toInstant(ZoneOffset.UTC));
+  }
+
+  private void setUpIdsAndUprns(CaseContainerDTO caseFromCaseService, CachedCase cachedCase) {
```

Comment:
Instead of this setup method which takes 2 cases it would be clearer if there was a single method with 3 args (the case to update and the uuid+uprn to use). 
The code using it would need to make 2 calls of course but it would be clearer in the test method about exactly what values are being set for each case. As it currently stands the use of the globals are buried and the reader needs to remember what the the method is doing to understand the test.

Reply:
I couldn't do a single method because the CachedCase gets updated slightly differently from the RM Case - the id is a String in a CachedCase but a UUID in an RM case (CaseContainerDTO).

Reply:
However, I have replaced the method with 2 methods that show what caseId and uprn values are being used more clearly, as you suggest, as I think that's a good idea.